templates/app.scaleft.yaml

```diff
@@ -0,0 +1,18 @@
+identities:
+- node:
+    name: app_scaleft
```

Comment:
usually scaleft_app (so all scaleft sort together)

Seems like a legitimate company that okta acquired. We should always check to make sure the domain owner is legitimate

Reply:
i made sure to check if they were legitimate before making a template for them